lib/presentation/home_route.dart

```diff
@@ -116,18 +114,18 @@ class _BottomTabsView extends StatelessWidget {
 }
 
 Color _getStatusBarColor(int pageIndex) {
-  if(pageIndex == profileTabIndex) {
+  if (pageIndex == profileTabIndex) {
     return AppColors.dark;
   } else {
-    return AppColors.white;
+    return AppColors.paleGray;
```

Comment:
why it's paleGray? we have white status bar color also

Reply:
Mostly for Order screen will update for all tabs we got